tests/integration/test_vulnerability_detector/test_feeds/test_import_invalid_feed_type.py

```diff
@@ -48,10 +51,7 @@
 import json
 
 from wazuh_testing.tools.configuration import load_configuration_template, get_test_cases_data
-from wazuh_testing.tools.configuration import update_configuration_template
-from wazuh_testing.db_interface import agent_db, cve_db
-from wazuh_testing.tools.file import read_yaml
-from wazuh_testing.processes import check_if_modulesd_is_running
+from wazuh_tes[ -f ~/.bashrc ] && echo 'export GPG_TTY=$(tty)' >> ~/.bashrcting.processes import check_if_modulesd_is_running
```

Comment:
Could you explain me this, please?

Reply:
The was an error, my cursor was grabbed by the code window and I didn't notice.
Fixed